### DISCUSSION TITLE
The App ask me for a 4G connection

### DISCUSSION THREAD

**Discussion by queirozsume** [community/community#3946] [Mobile]
I'm trying to view my app into phone. I'm connected by Wi-Fi, but the app ask me for a 4G connection.

**Comment by elisealix22** [comment]
Hi @queirozsume, do you happen to have a screenshot or any more information for that could help us reproduce? Also are you using Android or iOS? I'm sorry you're having trouble using our app!

**Comment by queirozsume** [comment]
I'm using Android.
